website/public/js/controllers/notificationCtrl.js

```diff
@@ -58,9 +58,11 @@ habitrpg.controller('NotificationCtrl',
 
     $rootScope.$watch('user._tmp.drop', function(after, before){
       // won't work when getting the same item twice?
-      if (after == before || !after) return;
-      $rootScope.playSound('Achievement_Unlocked');
-      if (after.type !== 'gear') {
+      if (_.isEqual(after, before) || !after) return;
```

Comment:
It wasn't working, in mysterious fashion. I would see the `if` statement blow right by, despite both `after` and `before` being, e.g. {type:'Quest',key:'atom1'}. Changing it to `_.isEqual` stopped that happening, eliminating the double-notification bug.